### ISSUE TITLE
146 addresses, carefully executed airdrop farming scheme from fresh wallets

### ISSUE THREAD

**Issue by stabiloswano** [safe-global/safe-user-allocation-reports#389]
# Related Safe Addresses
```
0x58ad3f6263ec5a080f52668e0114ab3c7dc4d5f4
0xa75882fa2b637c1f1ec133ad17c569bc0ade4a28
0x83ee0333cfb3eb4aa14188c099d3c9e5ca3dadec
0xa4daadfba6264eb98232acb28ce0cff3aadc6d36
0x877b3f0dfa0a37e27b2194e90ab40343f0792e79
0xcae4159d70462ba56a949f5cab922ff6b9c4ff54
0x2f6049f03eb16a49d3c6391801338fcefacbcdc1
0x8a0068e16d4203bbd469f1efcb7119d72352874d
0x9da8bb371044b1e85a6078f002163304aa74aa1c
0x4387f1cbacabf6a514e8653eacc086998c14ea41
0xf7d912893fcf833a287f3a6b018f0be09b597471
0xd120adababe6b959d28bfcb7114c705c4409d025
0x7b3c6405338fdc09e2cb31c02cbe50c2b28121b2
0x2dea964e4f37d3cd6a36a390ffd74a49d304eb82
0x632e13064933ae40d56942d1bc15c240c2ae4d61
0x45fe72a0f491358aff381acbdbe0a66b3bc34397
0xa5e649024678e7e4e7b196a08e8e932b2fa339c2
0x78e9dcbab5f8fb0cabba03606d7897c81c6ac291
0x51cdf09d77421643f1f9710d991d5430ac12ef2e
0xfce70ff33d2607296b117a29f964bdfcb2c40f02
0x85a97318db245ee5aaad85ca1280014c78171670
0x9039319d752b1fca07d2f4b2bac26a78b5bca6f2
0xbe0bfb0b40cbcf812ea50495ecada7f8f43e3b84
0xae9b84ac59d6e5fe0999a7a6937a88a46c93f3bc
0x3ac66a76b15e2aed55915968dd8e09235dc45b24
0xbaa0e3436a34e1419733f56dc08f624f3c9d4df1
0x6d6757abd430283ee7930cb5f739d1e159d02892
0xcd585509f7e0c59179a1396a78d17d387802a9ae
0x3127f62a428f4e3e9995675cfe96f456730be067
0x58716bab52f1d0dbdf59fce94c2e93497ce0d650
0x8885b4945b98d313f55f3998ec69ed9d3fd3c3ae
0xdc77d0d8371219a4dc2a2da0d402714fad7d0bf4
0xedcbd9343b0d1fc9fb577c41476549cc942edd44
0x314b40bb168f58d501e79db8fab7410071f61814
0xfc9c2479d6c79664b3ef7b3e79facfaa11b9c296
0xefe704c69e7ea27798c6bea2f28105e83b6f1cb2
0x2f2ba7d20c439ecc3859aced29d0b3c1e1dca993
0xa2765d60b6e645a78a2976286a5e246f3955a5d4
0x9e1c2e8619eeceb8c0a03901585ac6d268b49cca
0x7c4a07fc965ffd8404441cab2b1bcbba3fdad0fa
0x576a664592d5a8041bf36de62f0dc68dd2538d26
0xccc74799f84c856f7e8c79f98a32e4d5fa8a604e
0x06a5308356571070ecea24a990091b1caba3268d
0x026a5311f67b988808e94083bb14138daa012b6f
0x7c3522b0db0b58ae6a88217745115b7034e3a0ed
0x63a50db7006912a1cbace03f17186ce2260fb8b9
0x3eaf8be99e07b3b1a9fc9b5030eaf0645801993c
0x91cdf1b534beb56ceaef0e0a79de41c940c59ddc
0x170984c99c187b86d136a21ff989de075a9c28e8
0x3e13719032917e354275c9160d2da83b923f6125
0x58406fe5212a84093067d4903fe53fba33eb90b1
0x0ff3854632becaf8afa10ba18c775b55be3d54b4
0x291df36ecedf9b804729ff760e261dea3854bfb7
0x7354a266f4fa66d991b30e6402c499e9d17f836b
0x820ff466d10c036363ab6933d9e8653a3118f56a
0x969a1ad9f4e943904eb771bd60515bcf7e0d8775
0x7839ab0ffcefb6dc26fd661cab2d756351d1f3ab
0x9a6976274532883d462e376753fdb2e4c6fbe680
0xc975c94b6bbdbd2756240a4f2f583ee64a4db9e0
0x65704cb168ed6b1569fe466d3e21fc71606ec534
0x49c0c92be50e1353e8d99aa076ca8f8d994a37c2
0x7be985b997cc84110328289c84c513a0e51e7ab9
0x5fabbcd93604f9bbb4f38b313577c1200923ae06
0xdb23090ae36e5677854b8a2414872829808d89bf
0x9836d78c87f78f1f43f454d7d3af0141a6d75f5a
0x23d454d30caa725041f2b5fe36285e76e3361e27
0x4412beedc64f5b1ac6b7c016c03363a6f575d0ff
0x9751ab834f1747797c45483c75a9367667e3ebd1
0x4a26b9d2122830461375aeeb2adfe012565e1d81
0x6eec0c6ac830c335bf6a729bc906a3ba803a4188
0x502e033dea88ddeade8821e956d3561d01bd50cd
0x995e2bbc9a4195d08006f4f6b4b4b80a3efc4d24
0x967e9779614d67af4f43020d7f72b7c1693fd36c
0x9202b1a266b3ce79a68a774534fdedb58bdde34a
0x57eff7cf7bf0154544b295cea5387ea870d49a9c
0xf5b025c7f2c42dc8c1572e10ab6b515d37c8dce6
0x5168362bf8506dcd2023ebe906b34d44dc824cab
0xf0555e11de62206d6985227c7b56c222f5b6a434
0xb74268ccd6437e52ccd2ec0b333bbaa744f5e821
0x323c39b651b7949703d600359da0bf0c5b396589
0x89dfe63274439ca32a0a132088381006ddb10259
0x3ba7c3648af1e61982098a2dac55a8f6b508ba47
0x11b82db7f59727b77ba0136ae69bdbe7e095c9fa
0xd1e7a84f4e0949c744960914e8b06ac95edb14a9
0xb16e88c792210a601b16523ce233b39b71f61538
0x805ca2eeba31f315269a4f1f9981446ce17c27b6
0xd0a9a90a53e914049acce65d2f1bb44797bd07ae
0x46c78a1eb1fa3c441306ccc94fe633d8db6769eb
0x06c2bc54de2b92e1f533aa189edb9f7ed805d05b
0xbe86d838cef84ba8da69de7ef76d8df42cde645e
0x3e9516d4ec7f1d187ad9196132b4f118d771c979
0xd4f419b75888021f0bd9a74d56e4fe0f81ddfdb0
0x4dc75b1cba0c317c101cf6b03cc643bb8b0bc49b
0x7543e222602d25a9cd83a49fcb8156a7982869a2
0x2c4e2ea67941ba82d99d0769c45edeef46b00356
0xd7f6026c718523f520ec194746710a2303bc34f6
0x62424ffcb2f989b690c767ba19f4b1659dca0dca
0xd4d9261a7bf3bc9a8bc6f0b2ef859598dadb0cb3
0xe25257a8794936ca3037a0274e29a499674002f1
0x38276ef08893a5db2c9389e6e52360e172139e48
0x59f3b6aff0122327ac5847341ec3615741bfb533
0x9742a2d39619ebe90cf2f9e85b3e02bd0afcd210
0xf63f22dc990e97cdcd7ad46a67f0240d666a59ae
0x172e55241869c004d73f86d7e87415822bd9a828
0xe42d5f72e6edcee6f8d087ea50622a98e7f510ff
0xc92423310fbff3d77d4dd91c99f6e668ed42e80a
0xb0013c6d2ef9bc3675cf5f8dbfc7c0ed13150ccf
0x0c542b53ecff2f2ed391db0626e84c7479225ab4
0x75ecbaed6d5647890e947808054a25c6d96a5ec1
0xe093f1a6a2e425e4692095fb9b0a0264b89bdfc4
0xacfd31497855673c5dd0af7e4023747c1deea10f
0x1aba02ecf2f2060b02653099a2ab9b5750ec591b
0xbe75f40adebdb2a1802c623240c4a99ca26a28a8
0xddf57cb4f223fdb8ddff03e05bf35255d6a518b6
0xc02caf142ca5fb84ad60883c47d20159bbf0f131
0x86516b9e9111e57930e5a3e5d3ce30b4eadf38fe
0xc1f4ad4212207436a8bd3635b3f5bcda9731a7a0
0x5731ed586b64e5bc929faf5e324b7bf6d0e300ed
0x29ca4086ff0a8ba7a1af96053f29cb660deea5e5
0xe2504416481a2aa96c2010a456dc918823337ef8
0x15b56c1af9918e5ec3ad0bc8bdd8a2e55b80a431
0x973c7c67b3bf62d8864aa94fe79aa6de94930fff
0xa3d0bcb6b7c6d321f67ae839e9feed04e7fafed5
0xdf3928384750b40fe6f21a73a5e236c44cf2b485
0x2defd07615810e5726c7272ba4361e89fe666fd2
0x74520bebbf0ba5be15373621845e51045387a227
0x03efc1ebb6b3ebd280addfea0c7778bf203c188c
0x0e77fe049f400a5c2d7598fa133d1d636065b6ed
0x8f105799a3adc9e012d38921e8c0467f6d86dcd0
0xca33124e8b4b3b7a530c8073d221658c3d10bb6d
0x6d0e9e5562500717323a7ccef6551476766d27b7
0x04b3d94dc72e01ff320e601523ddcbf6cd6915ab
0x230006ed9586eaf550d74b8120018b9d117c62cd
0x41f6ce42940ffc7295bf36ea90d3ad1a1902e37c
0xf4a863c9d6598005ccdab4f5f0b711ea029dcc58
0x9ca9a16ed1079f544410aa5996ea6c9090f1546c
0xecc9db98051882b48229c0d607f2ded829674c51
0x0341bd434f39abab044adfc5f89634e818e1c977
0x1b8e8790467d4f994d5ec519c73ba95d2af06279
0x73a74cb40a2242a9108f4c4c9149ff9a310a0e53
0xc9510f3057522db12277c492bb60137a149deb11
0x3fce98809bef3782b1c7fd4674f3ea856cd5f246
0xe7d00ca7c8cb108d14b308e4d4cff98a2070e941
0xd31e963dda3d815e6fdfe7d274fbf1e603a96914
0x42b9ed2779bd96fae33cf9cadc84ac27019e1a15
0x7da590f6428152112bd3927fb3508bfce65b97ad
```
# Reasoning

1. Suspected airdrop farmer starts funding wallets from FTX (0.0115-0.01549) on 2022-08-04 02:16:59 UTC and by 2022-08-04 02:43:14 has funded 146 fresh addresses.

2. Then between 2022-08-04 18:23:12 and 2022-08-04 18:32:58 105 of the fresh funded addresses create a safe all in separate transactions, in 4 different blocks all using exactly 8 gwei as gas price. 	
   Later on the same day between 21:50:52 and 21:51:19 the remaining 41 fresh addresses continue creating safes in transactions, in 2 different blocks, all using exactly 6 gwei.

3. All 146 freshly created safes are funded from FTX (0.0045-0.00649) between 2022-08-05 00:38:07 - 01:01:33.

4. Then between 2022-08-05 06:22:43 - 08:46:11 all the creator addresses of safes start signing transfers from safe to completely fresh wallets (separate for every safe).
   For example 0xe0ea1531163be158f47b948a274187d3bbd3ffc1 (creator of 0x230006ed9586eaf550d74b8120018b9d117c62cd) executes transfer of 0.00232 ETH to 0x38f8da5c4727e7148d0f766bd83d245b7243b728.
   
5. Then during 2022-08-06 09:44:28 - 10:10:06 all the addresses that received ETH in previous step, return part of it to the safe that transfered it.

6. After more than 3 weeks of no transactions in any address of safe, all addresses transfer some eth back to safes, during 3 time periods 2022-08-28 17:35:06-17:34:53, 2022-08-28 21:15:20-2022-08-28 21:15:30 and 2022-08-29 01:59:37-01:59:52)

All the safes have exactly 4 events:
1. Funded from FTX wallet.
2. Transfer ETH to fresh wallet.
3. Receive some ETH back from the wallet in 2.
4. Reveive some ETH back again from the wallet in 2.
	
All the safe creating addresses have exactly 3 events (except 0xaf73951c4b13af0f363b6a6a3b50e304723c6f32 which was funded from FTX 2 times):
1. Funded from FTX wallet.
2. Create Safe.
3. Exec transaction in Safe.

146 addresses + safes performing all the same actions in tight time-windows clearly indicates that this is extensive and well-executed air drop farming performed by 1 person.

# Methodology
Pulled data from Etherscan and detected an anomaly where suspiciously high amount of new safes were created in short time frame.

Source datasets:
Full transaction log from creators of all mentioned safes:
[sybil_creator_full_transaction_log.csv](https://github.com/safe-global/safe-user-allocation-reports/files/9551931/sybil_creator_full_transaction_log.csv)
Full transaction log from all mentioned safes:
[sybil_safe_transaction_log.csv](https://github.com/safe-global/safe-user-allocation-reports/files/9551909/sybil_safe_transaction_log.csv)
Safe address and safe creator address:
[sybil_safe_creators.csv](https://github.com/safe-global/safe-user-allocation-reports/files/9551930/sybil_safe_creators.csv)

# Safe Address

0x3B0f3e067cc24F40C28CDd8f36C8243F5d7DE72A

**Comment by stabiloswano** [comment]
Failed with the search and thought these were unreported. These addresses are the same as earlier reported in https://github.com/safe-global/safe-user-allocation-reports/issues/306#issue-1368160691

**Comment by clcoin** [comment]
Hi, I accidentally found out a wallet of mine was included in this report. Let me first express my respect, this was indeed quality work and really impressive. And I also would like to point out that issue #306 was actually plagiarizing from this one, and you will know that if you check its edit history. What a shame on that reporter, and kudos to your gesture on your comments under both reports.

However, though your reasoning might look plausible, this was not airdrop farming by 1 person. I believe most of these addresses (I know a few) likely belong to members of an elite DAO I'm in, and we had zoom meetings to hangout and talk about crypto investments. These were mostly created and operated during a few recent zoom meetings with step by step instructions demonstrated by our founder, and we just followed. One of our members had a hot wallet hacked and lost all his tokens and NFTs which were worth more than $100k, so we felt we needed some safer wallets. But unfortunately many of us don't have a reliable channel to get good hardware wallets, so the founder proposed to use Safe wallets signed by a few new hot wallets as a safe way for asset storage, which is much safer than putting assets in a hot wallet. These were our practice Safes, and our upcoming membership NFT would also be sent to these Safes. To encourage participation, he used a bit of our donated DAO fund to pay for gas for Safe creation, and later transactions to make sure we can actually sign transactions using our hot wallets.

That's how I learned about Safe and I think it's great. It's a pleasant surprise that we may even get some SAFE tokens, but when I was checking I found this report that made the incorrect claim. Getting the SAFE airdrop was clearly not our purpose, and to be honest I think most of us don't even care about it. When I first heard about the airdrop eligibility, I was planning to donate it to our DAO fund anyways, same as many other DAO members would do. But it is an insult to me and other members to be called a "sybil" or "airdrop farmer", and that's what we really care about. We are just some individual investors trying to learn some crypto basics, and also we don't want to stop using Safe just because we are called as "sybils" like this. Hope you can understand our position.

Other than that part, again, I respect your work here and it definitely has higher quality than most others I spot checked, and I respect even more the nobleness you showed after you thought issue #306 was earlier (which was clearly not the case). Please don't be disappointed, and keep up with the good work!

**Comment by LakePatriciab** [comment]
1. The post was created on September 9 and the content is empty
2. On September 13th, GMT+0, before 4:00, the content of your post was still empty
3. #389 was posted on September 12th, GMT+0, 21:44
4. On September 13th, GMT+0, 4:11, you copied the method of #389 and posted the content
**This is the real witch act, it's shameful**

**Comment by FrostGabrielgav** [comment]
Someone in my DAO alerted me that I had been reported as a witch. (By the way, it took me quite a while to sign up a github account). I want to say, is it  just because someone taught us who can't buy reliable ledger in some area in the world and then teach us the reason for transferring money? What if I prove later that I'm not a witch? This is ridiculous. In addition, at the time of time, I didn’t know that safe had an airdrop. Didn’t the snapshot happen very early?

**Comment by tschubotz** [comment]
Thanks for the report. We've found the following to be related to airdrop farming:

```
0x58ad3f6263ec5a080f52668e0114ab3c7dc4d5f4
0xa75882fa2b637c1f1ec133ad17c569bc0ade4a28
0x83ee0333cfb3eb4aa14188c099d3c9e5ca3dadec
0xa4daadfba6264eb98232acb28ce0cff3aadc6d36
0x877b3f0dfa0a37e27b2194e90ab40343f0792e79
0xcae4159d70462ba56a949f5cab922ff6b9c4ff54
0x2f6049f03eb16a49d3c6391801338fcefacbcdc1
0x8a0068e16d4203bbd469f1efcb7119d72352874d
0x9da8bb371044b1e85a6078f002163304aa74aa1c
0x4387f1cbacabf6a514e8653eacc086998c14ea41
0xf7d912893fcf833a287f3a6b018f0be09b597471
0xd120adababe6b959d28bfcb7114c705c4409d025
0x7b3c6405338fdc09e2cb31c02cbe50c2b28121b2
0x2dea964e4f37d3cd6a36a390ffd74a49d304eb82
0x632e13064933ae40d56942d1bc15c240c2ae4d61
0x45fe72a0f491358aff381acbdbe0a66b3bc34397
0xa5e649024678e7e4e7b196a08e8e932b2fa339c2
0x78e9dcbab5f8fb0cabba03606d7897c81c6ac291
0x51cdf09d77421643f1f9710d991d5430ac12ef2e
0xfce70ff33d2607296b117a29f964bdfcb2c40f02
0x85a97318db245ee5aaad85ca1280014c78171670
0x9039319d752b1fca07d2f4b2bac26a78b5bca6f2
0xbe0bfb0b40cbcf812ea50495ecada7f8f43e3b84
0xae9b84ac59d6e5fe0999a7a6937a88a46c93f3bc
0x3ac66a76b15e2aed55915968dd8e09235dc45b24
0xbaa0e3436a34e1419733f56dc08f624f3c9d4df1
0x6d6757abd430283ee7930cb5f739d1e159d02892
0xcd585509f7e0c59179a1396a78d17d387802a9ae
0x3127f62a428f4e3e9995675cfe96f456730be067
0x58716bab52f1d0dbdf59fce94c2e93497ce0d650
0x8885b4945b98d313f55f3998ec69ed9d3fd3c3ae
0xdc77d0d8371219a4dc2a2da0d402714fad7d0bf4
0xedcbd9343b0d1fc9fb577c41476549cc942edd44
0x314b40bb168f58d501e79db8fab7410071f61814
0xfc9c2479d6c79664b3ef7b3e79facfaa11b9c296
0xefe704c69e7ea27798c6bea2f28105e83b6f1cb2
0x2f2ba7d20c439ecc3859aced29d0b3c1e1dca993
0xa2765d60b6e645a78a2976286a5e246f3955a5d4
0x9e1c2e8619eeceb8c0a03901585ac6d268b49cca
0x7c4a07fc965ffd8404441cab2b1bcbba3fdad0fa
0x576a664592d5a8041bf36de62f0dc68dd2538d26
0xccc74799f84c856f7e8c79f98a32e4d5fa8a604e
0x06a5308356571070ecea24a990091b1caba3268d
0x026a5311f67b988808e94083bb14138daa012b6f
0x7c3522b0db0b58ae6a88217745115b7034e3a0ed
0x63a50db7006912a1cbace03f17186ce2260fb8b9
0x3eaf8be99e07b3b1a9fc9b5030eaf0645801993c
0x91cdf1b534beb56ceaef0e0a79de41c940c59ddc
0x170984c99c187b86d136a21ff989de075a9c28e8
0x3e13719032917e354275c9160d2da83b923f6125
0x58406fe5212a84093067d4903fe53fba33eb90b1
0x0ff3854632becaf8afa10ba18c775b55be3d54b4
0x291df36ecedf9b804729ff760e261dea3854bfb7
0x7354a266f4fa66d991b30e6402c499e9d17f836b
0x820ff466d10c036363ab6933d9e8653a3118f56a
0x969a1ad9f4e943904eb771bd60515bcf7e0d8775
0x7839ab0ffcefb6dc26fd661cab2d756351d1f3ab
0x9a6976274532883d462e376753fdb2e4c6fbe680
0xc975c94b6bbdbd2756240a4f2f583ee64a4db9e0
0x65704cb168ed6b1569fe466d3e21fc71606ec534
0x49c0c92be50e1353e8d99aa076ca8f8d994a37c2
0x7be985b997cc84110328289c84c513a0e51e7ab9
0x5fabbcd93604f9bbb4f38b313577c1200923ae06
0xdb23090ae36e5677854b8a2414872829808d89bf
0x9836d78c87f78f1f43f454d7d3af0141a6d75f5a
0x23d454d30caa725041f2b5fe36285e76e3361e27
0x4412beedc64f5b1ac6b7c016c03363a6f575d0ff
0x9751ab834f1747797c45483c75a9367667e3ebd1
0x4a26b9d2122830461375aeeb2adfe012565e1d81
0x6eec0c6ac830c335bf6a729bc906a3ba803a4188
0x502e033dea88ddeade8821e956d3561d01bd50cd
0x995e2bbc9a4195d08006f4f6b4b4b80a3efc4d24
0x967e9779614d67af4f43020d7f72b7c1693fd36c
0x9202b1a266b3ce79a68a774534fdedb58bdde34a
0x57eff7cf7bf0154544b295cea5387ea870d49a9c
0xf5b025c7f2c42dc8c1572e10ab6b515d37c8dce6
0x5168362bf8506dcd2023ebe906b34d44dc824cab
0xf0555e11de62206d6985227c7b56c222f5b6a434
0xb74268ccd6437e52ccd2ec0b333bbaa744f5e821
0x323c39b651b7949703d600359da0bf0c5b396589
0x89dfe63274439ca32a0a132088381006ddb10259
0x3ba7c3648af1e61982098a2dac55a8f6b508ba47
0x11b82db7f59727b77ba0136ae69bdbe7e095c9fa
0xd1e7a84f4e0949c744960914e8b06ac95edb14a9
0xb16e88c792210a601b16523ce233b39b71f61538
0x805ca2eeba31f315269a4f1f9981446ce17c27b6
0xd0a9a90a53e914049acce65d2f1bb44797bd07ae
0x46c78a1eb1fa3c441306ccc94fe633d8db6769eb
0x06c2bc54de2b92e1f533aa189edb9f7ed805d05b
0xbe86d838cef84ba8da69de7ef76d8df42cde645e
0x3e9516d4ec7f1d187ad9196132b4f118d771c979
0xd4f419b75888021f0bd9a74d56e4fe0f81ddfdb0
0x4dc75b1cba0c317c101cf6b03cc643bb8b0bc49b
0x7543e222602d25a9cd83a49fcb8156a7982869a2
0x2c4e2ea67941ba82d99d0769c45edeef46b00356
0xd7f6026c718523f520ec194746710a2303bc34f6
0x62424ffcb2f989b690c767ba19f4b1659dca0dca
0xd4d9261a7bf3bc9a8bc6f0b2ef859598dadb0cb3
0xe25257a8794936ca3037a0274e29a499674002f1
0x38276ef08893a5db2c9389e6e52360e172139e48
0x59f3b6aff0122327ac5847341ec3615741bfb533
0x9742a2d39619ebe90cf2f9e85b3e02bd0afcd210
0xf63f22dc990e97cdcd7ad46a67f0240d666a59ae
0x172e55241869c004d73f86d7e87415822bd9a828
0xe42d5f72e6edcee6f8d087ea50622a98e7f510ff
0xc92423310fbff3d77d4dd91c99f6e668ed42e80a
0xb0013c6d2ef9bc3675cf5f8dbfc7c0ed13150ccf
0x0c542b53ecff2f2ed391db0626e84c7479225ab4
0x75ecbaed6d5647890e947808054a25c6d96a5ec1
0xe093f1a6a2e425e4692095fb9b0a0264b89bdfc4
0xacfd31497855673c5dd0af7e4023747c1deea10f
0x1aba02ecf2f2060b02653099a2ab9b5750ec591b
0xbe75f40adebdb2a1802c623240c4a99ca26a28a8
0xddf57cb4f223fdb8ddff03e05bf35255d6a518b6
0xc02caf142ca5fb84ad60883c47d20159bbf0f131
0x86516b9e9111e57930e5a3e5d3ce30b4eadf38fe
0xc1f4ad4212207436a8bd3635b3f5bcda9731a7a0
0x5731ed586b64e5bc929faf5e324b7bf6d0e300ed
0x29ca4086ff0a8ba7a1af96053f29cb660deea5e5
0xe2504416481a2aa96c2010a456dc918823337ef8
0x15b56c1af9918e5ec3ad0bc8bdd8a2e55b80a431
0x973c7c67b3bf62d8864aa94fe79aa6de94930fff
0xa3d0bcb6b7c6d321f67ae839e9feed04e7fafed5
0xdf3928384750b40fe6f21a73a5e236c44cf2b485
0x2defd07615810e5726c7272ba4361e89fe666fd2
0x74520bebbf0ba5be15373621845e51045387a227
0x03efc1ebb6b3ebd280addfea0c7778bf203c188c
0x0e77fe049f400a5c2d7598fa133d1d636065b6ed
0x8f105799a3adc9e012d38921e8c0467f6d86dcd0
0xca33124e8b4b3b7a530c8073d221658c3d10bb6d
0x6d0e9e5562500717323a7ccef6551476766d27b7
0x04b3d94dc72e01ff320e601523ddcbf6cd6915ab
0x230006ed9586eaf550d74b8120018b9d117c62cd
0x41f6ce42940ffc7295bf36ea90d3ad1a1902e37c
0xf4a863c9d6598005ccdab4f5f0b711ea029dcc58
0x9ca9a16ed1079f544410aa5996ea6c9090f1546c
0xecc9db98051882b48229c0d607f2ded829674c51
0x0341bd434f39abab044adfc5f89634e818e1c977
0x1b8e8790467d4f994d5ec519c73ba95d2af06279
0x73a74cb40a2242a9108f4c4c9149ff9a310a0e53
0xc9510f3057522db12277c492bb60137a149deb11
0x3fce98809bef3782b1c7fd4674f3ea856cd5f246
0xe7d00ca7c8cb108d14b308e4d4cff98a2070e941
0xd31e963dda3d815e6fdfe7d274fbf1e603a96914
0x42b9ed2779bd96fae33cf9cadc84ac27019e1a15
0x7da590f6428152112bd3927fb3508bfce65b97ad
```

**Comment by clcoin** [comment]
@tschubotz I just got email notification that this report has been accepted as valid. This is particularly sad as you just violated the principle stated in your own [announcement](https://forum.gnosis-safe.io/t/community-challenge-identify-airdrop-farmers/847) "Methodology that has a non-negligible chance of eliminating legitimate users will not be considered. When in doubt, we opt for “this is legit usage”." Basically a group of elite DAO members following live demo trying to learn how to use Safe as a safe asset store is not "legit usage" in your opinion. While at the same time, I saw some other reports clearly showing one address owning hundreds of Safes with little transactions are marked as invalid reports. Nice job.
I mentioned this report in our community earlier and most of us didn't care about the airdrop (ironically, aren't we "sybils"? lol) and we never expected it anyway, but many (including myself) were pissed off to be called "sybils". I think most of us would stop using Safe (some of us already created main Safe accounts separately other than these practice accounts created during our meetings), and we will for sure not distribute our membership NFT to these addresses now (glad we haven't done that yet). Goodbye Safe, sadly.
